### ISSUE TITLE
How to add "custom-element" tag to script

### ISSUE THREAD

**Issue by anjali89r** [vueuse/head#181]
Hi, 
I'm using Nuxt3 to create a web-story for my website and want to import the web-story script async in the head section of the page.
As per the [docs](https://amp.dev/documentation/guides-and-tutorials/start/visual_story/start_story), I need to add this to the head,
```
<script async custom-element="amp-story"
        src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
</head>
```
I was able to add the src and async, but can't seem to add custom-element. Can you please guide me on how to do it in vueuse head?
 
 Nuxt Page:

```
<script setup lang="ts">
useHead({
  title: 'My App',
  meta: [
    { name: 'description', content: 'My amazing site.' }
  ],
  bodyAttrs: {
    class: 'test'
  },
  script: [ { async: true,src: 'https://cdn.ampproject.org/v0/amp-story-1.0.js' } ]
})
</script>
```

**Comment by harlan-zw** [comment]
This will work, though `custom-element` isn't typed so you'll need to ignore the ts error. 

```ts
<script setup lang="ts">
useHead({
  title: 'My App',
  meta: [
    { name: 'description', content: 'My amazing site.' }
  ],
  bodyAttrs: {
    class: 'test'
  },
  script: [ { async: true, src: 'https://cdn.ampproject.org/v0/amp-story-1.0.js', 'custom-element': 'amp-story' } ]
})
</script>
```

I'll leave this issue open to type the `custom-element`

**Comment by anjali89r** [comment]
thanks @harlan-zw

**Comment by harlan-zw** [comment]
Hey @anjali89r 

Thanks for the issue, the type definitions are available in zhead 2.0.1 which will land in the next release of @vueuse/head, which should be released soon.
